distribution/neumann_alloc.json

```diff
@@ -0,0 +1,18 @@
+[
+	[
+		"5Cd7iTSbkuRqRJw791trBUZQq76Z4VPEuwyJwGpgW4ShzPvh",
+        2222222222222222223
+	],
+	[
+		"5CM2JyPHnbs81Cu8GzbraqHiwjeNwX3c9Rr5nXkJfwK9fwrk",
+		3333333333333333333
+	],
+    [
+		"5GcD1vPdWzBd3VPTPgVFWL9K7b27A2tPYcVTJoGwKcLjdG5w",
+        3333333333333333333
+	],
+    [
+        "5DhqAmCUoY7FiMCY7YpTyGT4fFpdXYmYjzzE9fHJwLDyBafZ",
```

Comment:
how did you split this into 4 accounts with these splits? i can't see how you're inserting the ratios in the chainspec.rs file below

Reply:
I made them up as tests. The real one will come later

Reply:
ah yah totally, i just meant i can't seem to find where you defined the ratios in the code